src/services/dashboard/handlers/templatizeCloudWatchDashboard.ts

```diff
@@ -55,7 +47,7 @@ export const handler = async (
       [region!]: "${env:REGION_A}",
       [service!]: "${self:service}",
```

Comment:
Instead of removing the linter's no non-null assertion, could we check the env vars after reading them in and `throw` if any are null/undefined? That should eliminate the need for the assertions here.